### PR TITLE
Filter out applications in the runtime based on application type

### PR DESCRIPTION
**Pull request by tgorgol**
**Description**

Changes proposed in this pull request:

- Filter out applications in the runtime based on application type

### CODE REVIEW

File: chart/compass/charts/director/templates/configmap-configuration.yaml [hunk 1]
```diff
@@ -7,4 +7,11 @@ metadata:
     app: {{ .Chart.Name }}
     release: {{ .Release.Name }}
 data:
-  {{- (.Files.Glob "config.yaml" ).AsConfig | nindent 2 }}
\ No newline at end of file
+  {{- (.Files.Glob "config.yaml" ).AsConfig | nindent 2 }}
```
[the remark this answers is not in the document]
Comment (tgorgol) [reply chart/compass/charts/director/templates/configmap-configuration.yaml:10]: We want to be able to override the filters on cluster, if we used only config.yaml as we did so far, we would have to open PRs witch changes to config, and any manual changes on cluster would be overwritten by CD with the defaults from chart.

File: docs/director/03-01-hiding-application-for-runtime.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Hiding Application for Runtime
+
+It is possible to configure Compass to not return Applications labeled with specific labels, in response to query `applicationsForRuntime` used by Runtimes. 
+
+## Configuration
+
+To configure the list of selectors which will be used to filter the list of returned Applications modify **applicationHideSelectors** parameter in the [`values.yaml`](https://github.com/kyma-incubator/compass/blob/master/chart/compass/charts/director/values.yaml) file. You can modify this list by providing appropriate overrides.
```
Comment (klaudiagrz) [inline docs/director/03-01-hiding-application-for-runtime.md:7]:
```suggestion
To configure the list of selectors used to filter the list of returned Applications, modify the **applicationHideSelectors** parameter in the [`values.yaml`](https://github.com/kyma-incubator/compass/blob/master/chart/compass/charts/director/values.yaml) file. You can modify this list by providing appropriate overrides.
```

File: docs/director/03-01-hiding-application-for-runtime.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+## Example
+```yaml
+applicationHideSelectors:
+  applicationType:
+    - "Test Application"
+    - "WIP"
+  applicationVersion:
+    - "alpha"  
+```
```
Comment (klaudiagrz) [inline docs/director/03-01-hiding-application-for-runtime.md:21]:
```suggestion


In this example, all Applications with the `applicationType: "Test Application"`, `applicationType: "WIP"`, or `applicationVersion: "alpha"` label are not visible for Runtimes.
```

Comment (klaudiagrz) [reply docs/director/03-01-hiding-application-for-runtime.md:21]: Please add such a sentence below.

File: docs/director/03-01-hiding-application-for-runtime.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# Hiding Application for Runtime
+
+It is possible to configure Compass to not return Applications labeled with specific labels, in response to query `applicationsForRuntime` used by Runtimes. 
+
+## Configuration
+
+To configure the list of selectors which will be used to filter the list of returned Applications modify **applicationHideSelectors** parameter in the [`values.yaml`](https://github.com/kyma-incubator/compass/blob/master/chart/compass/charts/director/values.yaml) file. You can modify this list by providing appropriate overrides.
+
+**applicationHideSelectors** should be a map of lists of **strings**, where map keys represent label keys and strings represent label values that will be used to filter the returned Applications.
```
Comment (klaudiagrz) [inline docs/director/03-01-hiding-application-for-runtime.md:9]:
```suggestion
The **applicationHideSelectors** parameter is a map that contains the lists of **strings**, where map keys represent label keys, and strings represent label values that are used to filter the returned Applications.
```

File: docs/director/03-01-hiding-application-for-runtime.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+Each label key can have multiple values assigned, if an Application is labeled with that key and any of the specified values, it will not be returned in `applicationsForRuntime` query. Multiple label keys can be configured.
```
Comment (klaudiagrz) [inline docs/director/03-01-hiding-application-for-runtime.md:11]:
```suggestion
Each label key can have multiple label keys and values assigned. If an Application is labeled with that key and any of the specified values, it is not returned in the `applicationsForRuntime` query.
```

File: docs/director/03-01-hiding-application-for-runtime.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+## Example
```
Comment (klaudiagrz) [inline docs/director/03-01-hiding-application-for-runtime.md:13]:
```suggestion
See the example of the labels specified in the **applicationHideSelectors** parameter:
```

File: docs/director/03-01-hiding-application-for-runtime.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
+# Hiding Application for Runtime
```
Comment (klaudiagrz) [inline docs/director/03-01-hiding-application-for-runtime.md:1]:
```suggestion
# Hide Applications from Runtimes
```

File: docs/director/03-01-hide-applications-from-runtimes.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+
+## Configuration
+
+To configure the list of selectors used to filter the list of returned Applications, modify the **applicationHideSelectors** parameter in the [`values.yaml`](https://github.com/kyma-incubator/compass/blob/master/chart/compass/charts/director/values.yaml) file. You can modify this list by providing appropriate overrides.
+
+The **applicationHideSelectors** parameter is a map that contains the lists of **strings**, where map keys represent label keys, and strings represent label values that are used to filter the returned Applications.
+
+Each label key can have multiple values assigned, if an Application is labeled with that key and any of the specified values, it will not be returned in `applicationsForRuntime` query. Multiple label keys can be configured.
+
+See the example of the labels specified in the **applicationHideSelectors** parameter:
```
Comment (klaudiagrz) [inline docs/director/03-01-hide-applications-from-runtimes.md:13]:
```suggestion
See the example of labels specified in the **applicationHideSelectors** parameter:
```

File: docs/director/03-01-hide-applications-from-runtimes.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+
+It is possible to configure Compass so that it doesn't return Applications labeled with specific labels in response to the `applicationsForRuntime` query used by Runtimes. 
+
+## Configuration
+
+To configure the list of selectors used to filter the list of returned Applications, modify the **applicationHideSelectors** parameter in the [`values.yaml`](https://github.com/kyma-incubator/compass/blob/master/chart/compass/charts/director/values.yaml) file. You can modify this list by providing appropriate overrides.
+
+The **applicationHideSelectors** parameter is a map that contains the lists of **strings**, where map keys represent label keys, and strings represent label values that are used to filter the returned Applications.
+
+Each label key can have multiple values assigned, if an Application is labeled with that key and any of the specified values, it will not be returned in `applicationsForRuntime` query. Multiple label keys can be configured.
```
Comment (klaudiagrz) [inline docs/director/03-01-hide-applications-from-runtimes.md:11]:
```suggestion
You can configure multiple label keys. Each label key can have multiple values assigned. If an Application is labeled with that key and any of the specified values, it is not returned in the `applicationsForRuntime` query.
```

**Pull request merged**